### PR TITLE
fix: improve advanced trading view

### LINKED ISSUE #1584: Improve advanced trading view
- Fix chart, trade and pool activities list loading
- Add link to order history
- Fix changing title on pool activity

### PR DESCRIPTION
**Pull request by berteotti**
# Summary

Fixes #1584

Improvements to advanced trading view.

  # To Test

1. On advanced trading view
- [ ] Check if trade and activity doesn't load anymore
- [ ] Check if swapping tokens changes pool activity title
- [ ] Link to explorer on order history if it's a swap
- [ ] No link to explorer on order history if it's a bridge swap
- [ ] Trading View chart loading

### CODE REVIEW

File: src/pages/Swap/AdvancedSwapMode/Chart/ChartLoader.component.tsx [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+import { AbsoluteWrapper } from './Chart.styles'
+
+const THREE_SECONDS = 3000
+
+export function ChartLoader({ symbol }: { symbol?: string }) {
+  const [isLoading, setIsLoading] = useState(true)
+
+  useEffect(() => {
+    setIsLoading(true)
+    const timeout = setTimeout(() => setIsLoading(false), THREE_SECONDS)
```
Comment (Diogomartf) [inline src/pages/Swap/AdvancedSwapMode/Chart/ChartLoader.component.tsx:13]: does this charLoader always take 3 seconds?

File: src/services/AdvancedTradingView/useAdvancedTradingView.hook.ts [hunk 2]
```diff
@@ -166,14 +166,6 @@ export const useAdvancedTradingView = () => {
     }
 
     fetchTrades()
-
```
Comment (Diogomartf) [inline src/services/AdvancedTradingView/useAdvancedTradingView.hook.ts:169]: why did we remove this code?

**Review by Diogomartf — APPROVED** [review]
Good job 😎

File: src/pages/Swap/AdvancedSwapMode/Chart/ChartLoader.component.tsx (hunk 1 above)
Comment (berteotti) [reply src/pages/Swap/AdvancedSwapMode/Chart/ChartLoader.component.tsx:13]: Yes, the chart needs to load at least 3 seconds

File: src/services/AdvancedTradingView/useAdvancedTradingView.hook.ts (hunk 2 above)
Comment (berteotti) [reply src/services/AdvancedTradingView/useAdvancedTradingView.hook.ts:169]: It wasn't working as expected... We need a more elaborated solution for this

**Comment by 0xVenky** [comment]
Seems to work as desired.  :)

**Review by 0xVenky — APPROVED** [review]
Approved

**Pull request merged**